### ISSUE TITLE
SA1514 incorrectly reported when preceded by preprocessor directive with line comment

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1223]
The following code reports SA1514, but it should not.

```csharp
#pragma warning disable RS1012 // Any comment.

/// <summary>
/// Summary
/// </summary>
static void MyMethod()
{
}
```

**Comment by alaendle** [comment]
Same problem exists for regions - e.g.

```
    #region Test

    /// <summary>
    /// Initializes a new instance of the <see cref="Form1"/> class.
    /// </summary>
    public Form1()
    {
      this.InitializeComponent();
    }

    #endregion
```

**Comment by sharwell** [comment]
@alaendle That one was reported in #1231.
